drivers/temperature/adt7420/adt7420.c

```diff
@@ -97,25 +97,80 @@ int adt7420_reg_read(struct adt7420_dev *dev,
  * @return 0 in case of success, -1 otherwise.
 *******************************************************************************/
 int adt7420_reg_update_bits(struct adt7420_dev *dev,
-			    uint8_t register_address, uint8_t mask, uint8_t value)
+			    uint16_t register_address, uint8_t mask, uint8_t value)
 {
 
-	uint8_t data_buffer[3] = {0, 0, 0};
 	uint16_t regval;
 
 	if (adt7420_reg_read(dev, register_address, &regval))
 		return -1;
 
 	regval &= ~mask;
 	regval |= value;
-	if (adt7420_is_spi(dev) && no_os_field_get(ADT7320_L16, register_address)) {
-		data_buffer[0] = regval >> 8;
-		data_buffer[1] = regval & 0xFF;
+
+	return adt7420_reg_write(dev, register_address, regval);
+}
+
+/***************************************************************************//**
+ * @brief Write a value of a register via SPI
+ * @param dev              - The device structure.
```

Comment:
There is no need for space here before the dash.

Reply:
Made the change